README.md

```diff
@@ -581,6 +583,46 @@ for other types of options.
 Many modifiers are polymorphic in this type argument, which means
 that they can be used with any builder.
 
+### Common Builder Patterns
+
+#### A list of values with a default
+
+There are a few ways we could parse multiple values for an option.
+
+We could use `some (strOption (long "arg-name" <> value "default"))`, which allows you to pass in values like this:
```

Comment:
actually this will blow stack (or never terminate if code wash stack safe) because when `some` or `many` is used the parser passed to them must fail at some point. because `value` is used it will never fail and many/some will also never fail and continue to try to parse until memory/stack blows up. instead `some (strOption (long "foo")) <|> NES.singleton "bzz"` can be used to accomplish that.